pkg/events/derive/hooked_syscall.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/aquasecurity/tracee/types/trace"
 )
 
+var SyscallsToCheck = make([]string, 0)
+var MaxSupportedSyscallID = 293 // Based on the __NR_syscalls value for the lowest supported version 4.18
```

Comment:
No way this is true.
BPF syscall, which was added in 3.17 has ID of 321 in amd64 arch.
Also, this value should probably be defined per architecture (in events/amd64.go and events/arm64.go)

Reply:
Fixed